### DISCUSSION TITLE
Align to left the dropdown content

### DISCUSSION THREAD

**Discussion by knupel** [shadcn-ui/ui#3360] [Q&A]
Hello, I'm beginner with shadcn, I wish use dropdown and set like I wish. My first wish it's try to align the content to left when I click on the button, not on the center like the example you give here : https://ui.shadcn.com/docs/components/dropdown-menu
<img width="888" alt="Capture d’écran 2024-04-04 à 18 23 19" src="https://github.com/shadcn-ui/ui/assets/3332269/f4362530-e4d5-4a18-9806-2228ddc59b39">

but more like in the `Tasks example`

<img width="484" alt="Capture d’écran 2024-04-04 à 18 24 07" src="https://github.com/shadcn-ui/ui/assets/3332269/17b7f877-7190-4cf2-9a8c-ca8f9818963e">

But the problem I don't understand where I must or I can set the dropdown content to manage that ?
I've supposed is in the `DropdownMenuContent` in the tailwind part, but not sure. Or maybe it's elsewhere.

I don't find example or tutorial to explain how to set the dropdown. So If you any idea to help me to set it, it's very welcome ! 

Thanks for your help and congratulation for this UI lib !

**Comment by Hubertneoworlder** [comment]
Hello @knupel 
You can align the dropdown content by setting align property of the `DropdownMenuContent` to `start`
```tsx
<DropdownMenu>
        <DropdownMenuTrigger className="outline-none">
           <span className="bg-muted rounded-full text-foreground h-[28px] w-[28px] inline-flex justify-center items-center">
               <FiPlus className="24px" />
           </span>
 </DropdownMenuTrigger>
<DropdownMenuContent align="start">
           <DropdownMenuLabel>Media</DropdownMenuLabel>
         <DropdownMenuSeparator />
         <DropdownMenuItem>Profile</DropdownMenuItem>
        <DropdownMenuItem>Billing</DropdownMenuItem>
    </DropdownMenuContent>
 </DropdownMenu>
```

**Reply by pranavarora724** [reply]
Thanks a Million 
You saved the day

**Comment by paulohernandez** [comment]
how to put that dropdownmenucontent to the bottom of the trigger ?

**Reply by Roman-Artemiev** [reply]
Hi, i find it, maybe it could help you https://github.com/huntabyte/shadcn-svelte/discussions/1098
